Model/Order/Builder.php

```diff
@@ -120,10 +115,7 @@ public function build(Order $order)
                 }
                 $nostoOrder->setOrderStatus($nostoStatus);
             }
-            $nostoBuyer = $this->buyerBuilder->fromOrder($order);
-            if ($nostoBuyer instanceof Buyer) {
-                $nostoOrder->setCustomer($nostoBuyer);
-            }
+            $nostoOrder->setCustomer(new Buyer());
```

Comment:
do we need to set customer as an empty Buyer? or can we just skip it and have it as null

Reply:
It's used later on to populate the customer id, so null would make it crash later on.